package.json

```diff
@@ -344,8 +344,9 @@
     "@brave/react-virtualized-auto-sizer": "^1.0.4",
     "@glif/filecoin-wallet-provider": "2.0.0-beta.12",
     "@ledgerhq/hw-app-eth": "6.24.1",
+    "@ledgerhq/hw-app-solana": "6.27.1",
     "@ledgerhq/hw-transport-webhid": "6.24.1",
-    "@solana/web3.js": "1.44.2",
+    "@solana/web3.js": "1.46.0",
```

Comment:
sorry but can we updated this to `1.47.1`? it should then fix one of the Socket.dev issues we've been seeing for this package